native/packages/react-native-bpk-component-card/src/BpkCard.js

```diff
@@ -91,17 +91,23 @@ const BpkCard = (props) => {
   const style = [styles.card];
   if (padded) { style.push(styles.cardPadded); }
   if (focused) { style.push(styles.cardFocused); }
-  if (userStyle) { style.push(userStyle); }
 
   return (
-    <TouchableHighlight
-      accessibilityComponentType="button"
-      underlayColor={colorGray50}
-      style={style}
-      {...rest}
-    >
-      <View style={styles.cardInner}>{children}</View>
-    </TouchableHighlight>
+    <View style={userStyle}>
+      <Touchable
+        style={styles.outerContainer}
+        useForeground
+        underlayColor={colorGray50}
+        accessibilityComponentType="button"
+        background={Platform.OS === 'android' ? TouchableNativeFeedback.SelectableBackgroundBorderless() : null}
```

Comment:
Android's `TouchableNativeFeedback` does not accepts styles, so the background white needs to be added to the inner view on Android